### PR TITLE
Change text editor, add publish date to article

### PR DESCRIPTION
**Pull request by bengourley**
Pretty much just bringing over updates from the Clock site

### CODE REVIEW

**Comment by bengourley** [comment]
Oh, also on this PR switched to jade template inheritance (vs. partials) for more Express 3 readiness. Only one instance of `partial()` left for the admin bar when logged in browsing the site.

**Comment by bengourley** [comment]
Merged Olivers comma changes, should be able to merge into your master now.

File: lib/utils/formHelper.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 module.exports.processors = {
   split: function(key, form) {
-    if ((form[key] !== undefined) && (typeof form[key] === 'string') && (form[key].length > 0)) {
```
Comment (serby) [inline lib/utils/formHelper.js:3]: Why did this need to be done? Could this break any other feature?

Comment (bengourley) [inline lib/utils/formHelper.js:3]: That wasn't my change, just got affected by running the tabs to spaces. It was introduced by your commit: https://github.com/bengourley/control/tree/1d1e991116f208bbd4d440f50965781fa996285f

**Pull request merged**